src/ixdat/techniques/reactor.py

```diff
@@ -0,0 +1,289 @@
+from .ms import MSMeasurement, SpectroMSMeasurement
+from ..measurements import Calibration
+from ..plotters.tpms_plotter import TPMSPlotter, SpectroTPMSPlotter
+from ..db import Saveable
+from ..data_series import ValueSeries
+import warnings
+
+
+class ReactorMeasurement(MSMeasurement):
+
+    default_plotter = TPMSPlotter
+    essential_series_names = ("temperature", "pressure")
+
```

Comment:
I don't actually see where the `Calibration` gets applied. Maybe there needs to be an `__init__` which has the line:
`self.add_calibration(ReactorCalibration())`

Reply:
Now it's explicit :)